src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionActions.ts

```diff
@@ -1,12 +1,23 @@
-import { createAction, createAsyncAction } from 'typesafe-actions';
+import {
+  createAction,
+  createStandardAction,
+  createAsyncAction
+} from 'typesafe-actions';
 import { ThunkAction } from 'redux-thunk';
 import { ActionCreator, Action } from 'redux';
+import { orthologueSpecies as sampleOrthologueSpecies } from '../../../../sample-data/orthologue';
```

Comment:
Instead of going four levels up the file tree, better use a path starting from `src`

Reply:
That's done.